plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/GenerateEnclaveMetadata.kt

```diff
@@ -40,6 +53,10 @@ open class GenerateEnclaveMetadata @Inject constructor(
         }
 
         val enclaveMetadata = EnclaveMetadata.parseMetadataFile(metadataFile)
+
+        mrsignerOutputFile.asFile.get().writeText(enclaveMetadata.mrsigner.bytes.toHexString().lowercase())
```

Comment:
Any idea of what is the different between `.get().asFile` vs `.asFile.get()`? I see we use both.

Reply:
Also, why lowercase? I think we generally used uppercase for these?

Reply:
Secure release process uses lowercase

Reply:
Secure release policy is an Azure thing, and since we already print these values in the attesation report toString and else in uppercase, we should be consistent and do the same here.